src/Tribe/Commerce/PayPal/Orders/Report.php

```diff
@@ -39,11 +39,11 @@ class Tribe__Tickets__Commerce__PayPal__Orders__Report {
 	 * @return string The absolute URL.
 	 */
 	public static function get_tickets_report_link( $post ) {
-		$url = add_query_arg( array(
+		$url = add_query_arg( [
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: ca91a6bc72f0b8fc9cbd5095ef86e6fb</sub>